apps/backend/src/authn/okta.strategy.ts

```diff
@@ -36,7 +36,7 @@ export class OktaStrategy extends PassportStrategy(Strategy, 'okta') {
         clientID: configService.get('OKTA_CLIENTID') || 'disabled',
         clientSecret: configService.get('OKTA_CLIENTSECRET') || 'disabled',
         callbackURL:
-          `${configService.get('EXTERNAL_URL')}/authn/okta/callback` ||
+          `${configService.get('EXTERNAL_URL')}authn/okta/callback` ||
```

Comment:
This seems inconsistent with everywhere else that we use EXTERNAL_URL:

https://github.com/mitre/heimdall2/blob/db2209395a9a20a21ec302559b95f711840ed057/apps/backend/src/authn/google.strategy.ts#L31

https://github.com/mitre/heimdall2/blob/db2209395a9a20a21ec302559b95f711840ed057/apps/backend/src/authn/gitlab.strategy.ts#L28